server/service/src/missing_program.rs

```diff
@@ -16,6 +17,11 @@ pub fn create_missing_master_list_and_program(
         description: MISSING_PROGRAM.to_string(),
     };
 
+    let missing_context = ContextRow {
+        id: MISSING_PROGRAM.to_string(),
```

Comment:
MISSING_CONTEXT?

Reply:
Its the same id as for MISSING_PROGRAM so I just used that...